test/core/interface.spec.js

```diff
@@ -154,7 +157,10 @@ describe('interface-ipfs-core tests', () => {
 
             if (typeof config === 'function') {
               cb = config
-              config = undefined
```

Comment:
Still need to set config to `null` here otherwise it'll still point to the callback function and you won't get your empty bootstraps ;)

Reply:
Added it back. I thought I already did, but my brain was wrong. Thanks for catching it!